code/modules/paperwork/desk_bell.dm

```diff
@@ -63,15 +57,29 @@
 			return TRUE
 		return FALSE
 
-// Deconstruct
+// Deconstruct and Anchor
 /obj/item/desk_bell/wrench_act(mob/living/user, obj/item/tool)
-	to_chat(user, "<span class='notice'>You begin taking apart the bell...</span>")
-	if(tool.use_tool(src, user, 5 SECONDS))
-		to_chat(user, "<span class='notice'>You disassemble the bell...</span>")
-		playsound(user, 'sound/items/deconstruct.ogg', 50, vary = TRUE)
-		new /obj/item/stack/sheet/metal(drop_location(), 2)
-		qdel(src)
-		return TRUE
+	. = TRUE
+	if(user.a_intent == INTENT_HARM && !in_inventory)
+		to_chat(user, "<span class='notice'>You begin taking apart the bell...</span>")
```

Comment:
Might want to make these visible messages instead so other people can see them.
```suggestion
		visible_message("<span class='notice'>[user] begins taking apart the bell...</span>", "<span class='notice'>You begin taking apart the bell</span>")
```

Reply:
I believe most of these only report it on completion, with the sound being the indicator that something is happening

Reply:
Actually, I'm probably wrong here